### PR TITLE
version 0.6.0

### PR DESCRIPTION
**Pull request by nejisama**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, 
details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
 
+## 0.6.0
+
+### New Features
+
+- New config field, "connection_idle_timeout" in Listener, mosn will close idle timeout connection. The default value is 90s. 
+- New function "Alertf" in ErrorLogger
+- Support get certificate from SDS mode
+
+### Refatcor
```
Comment (antJack) [inline docs/CHANGELOG.md]: typo: Refactor

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+
+### Performance Optimize
+
+- Optimize connection metrics
+- Optimize metrics prometheus output format
+- Optimize network IO
+
+### Bug Fix
+
+- Fix maybe panic when create logger concurrent
```
Comment (antJack) [inline docs/CHANGELOG.md]: Fix panic when creating logger concurrently

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## 0.6.0
+
+### New Features
+
+- New config field, "connection_idle_timeout" in Listener, mosn will close idle timeout connection. The default value is 90s.
```
Comment (antJack) [inline docs/CHANGELOG.md]: Add "connection_idle_timeout" field to Listener config, default 90s. Mosn will close idle connections when timeout.

File: docs/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,37 @@
...
+
+### New Features
+
+- New config field, "connection_idle_timeout" in Listener, mosn will close idle timeout connection. The default value is 90s. 
+- New function "Alertf" in ErrorLogger
+- Support get certificate from SDS mode
+
+### Refatcor
+
+- Refactor package upstream
```
Comment (antJack) [inline docs/CHANGELOG.md]: Refactor the upstream package

File: docs/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,37 @@
...
+  - Refactor the inner cluster structure
+  - UpdateHost updates all hosts every time, does not compute the difference any more
```
Comment (antJack) [inline docs/CHANGELOG.md]: UpdateHost does full update every time, instead of incremental update.

File: docs/CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,37 @@
...
+### Performance Optimize
+
+- Optimize connection metrics
+- Optimize metrics prometheus output format
+- Optimize network IO
+
+### Bug Fix
+
+- Fix maybe panic when create logger concurrent
+- Fix maybe panic when response and timeout are concurrent
```
Comment (antJack) [inline docs/CHANGELOG.md]: Fix panic when doing response and timeout simultaneously.

File: docs/CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## 0.6.0
+
+### New Features
+
+- New config field, "connection_idle_timeout" in Listener, mosn will close idle timeout connection. The default value is 90s. 
+- New function "Alertf" in ErrorLogger
+- Support get certificate from SDS mode
```
[the remark this answers is not in the document]
Comment (antJack) [reply docs/CHANGELOG.md]: Support getting certififactes via SDS

[the remark this answers is not in the document]
Comment (antJack) [reply docs/CHANGELOG.md]: Support getting certificates via SDS
